fabtests/prov/efa/src/rdm_remote_exit_early.c

```diff
@@ -203,6 +218,9 @@ int main(int argc, char **argv)
 		case LONG_OPT_POST_RX:
 			post_rx = true;
 			break;
+		case LONG_OPT_USE_RDMA_READ:
```

Comment:
You don't have to use such argument. You can query whether rdma read is available by calling `fi_getopt(... FI_OPT_EFA_EMULATED_READ, ...)`. If it is false, it means rdma read is available.